### DISCUSSION TITLE
Throttle Android focus change when user's holding the key

### DISCUSSION THREAD

**Discussion by msynowski** [react-native-tvos/react-native-tvos#537] [Feature Requests]
Hi.
Would it be possible to throttle changing focus on Android? 
Currently when you hold down the directional key, focus changes insanely fast, causing both performance and UX issues.

**Comment by douglowder** [comment]
We will look into this. It might be useful on both TV platforms. @yusufyildirim

**Reply by yusufyildirim** [reply]
I always feel like platform defaults are there for a reason but I can't deny this could be useful especially for the lists. So maybe we can provide a way. It shouldn't be hard to achieve. I'll think about this a bit more to justify the idea

**Reply by msynowski** [reply]
The defaults are also being overwritten by Google itself. 
Open preinstalled YouTube app, and try to hold an arrow key down

**Reply by yusufyildirim** [reply]
Of course, it's a product decision, they can do that. It's not an embedded feature of the platform itself (as I'm aware), that's what I'm saying. We want users to navigate around as fast as possible most of the time. For the rest, a mechanism like this could be helpful for sure. I have something in mind and will implement something container-basis. I won't be able to do it very soon though 😄
